.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,25 +7,30 @@ assignees: ''
 
 ---
 
-<!-- Thanks for helping us to improve k3s! We welcome all bug reports. Please fill out each area of the template so we can better help you.  ***You can delete this message portion of the bug report.*** -->
+<!-- Thanks for helping us to improve K3s! We welcome all bug reports. Please fill out each area of the template so we can better help you. Comments like this will be hidden when you post but you can delete them if you wish. -->
 
-**Version:**
-<!-- Provide the output from `k3s -v` -->
+**Environmental Info:**
+K3s Version: 
+<!-- Provide the output from "k3s -v" -->
 
-**K3s arguments:**
-<!-- Provide the flags used to install or run k3s server. -->
+Node(s) CPU architecture, OS, and Version: 
+<!-- Provide the output from "uname -a" on the node(s) -->
```

Comment:
```suggestion
<!-- Provide the output from `uname -a` on the node(s) -->
```

Reply:
I dont think this will work in comments. Once rendered in pretty/html view the comments will be hidden. Comments only seen while editing/creating an issue.